__mocks__/dimensions.mock.ts

```diff
@@ -185,8 +189,8 @@ export const mockDimensionResponse = {
           context: {
             reference: "Encounter/1a82fdc6-3d14-44c4-ab45-d65baa0062ae"
           },
-          effectiveDateTime: "2016-12-18T06:48:20+00:00",
```

Comment:
The `+00:00` asserts time zone in a way that can actually change which day it is for your local computer. That's why I removed it